#CREATE request parameters
string[] sound
--------------------------------
#CREATE response parameters
bool playing